test.sh

```diff
@@ -0,0 +1,41 @@
+PLATFORM=$1
```

Comment:
New script, works on windows and is dead simple to operate (or so I think), embracing our BASH overlords.